core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/ExoPlayerExtractorMediaSourceListener.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.source.ExtractorMediaSource;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+class ExoPlayerExtractorMediaSourceListener implements ExtractorMediaSource.EventListener {
```

Comment:
Same as others above. We no longer extend from `ExoPlayerEventForwarder` but instead implement the interface we care about directly from `ExoPlayer`.